packages/visualizations/src/components/Map/MapRender.svelte

```diff
@@ -82,12 +80,11 @@ TODO:
 
     function initializeResizer() {
         // Set a resizeObserver to resize map on container size changes
+        // TODO: Do we really want to reset to the initial bbox each time?
```

Comment:
I guess it's really up to us and open for discussion, my feeling is that this works well and I'm afraid that someone resizing his window can be lost on a map with a huge zoom but I'm not closed to remove that as well.

Reply:
I added this comment because after my last "findings", I know that in some situations, queryRenderedFeatures / querySourceFeatures will not return every shape, and typically, if there is a situation where we have both large and very small shapes, we'll may start initially at a zoom where we won't get the smaller shape. So, we may compute a maximum zoom that will ignore smaller shapes and will actually be too wide.
But:
- This will happen 0.001% of the time?
- Even if this happen, it would require a very very very large difference of scale to give us a maximum zoom that is really too far from the small shape
- In other situations, it's rather cool in terms of usability

So I'm tempted to keep it as well and see if there's really a real-life situation that occurs. If it happens I'm not sure what we can do, but I think it won't...